test/behavior/LinkToken.behavior.ts

```diff
@@ -0,0 +1,262 @@
+import { Wallet, ContractFactory, Contract } from 'ethers'
+import { BigNumber } from 'ethers/utils'
+import { setup, matchers } from '@chainlink/test-helpers'
+import * as h from '../helpers'
+
+export const shouldBehaveLikeLinkToken = (
+  linkTokenFactory: ContractFactory,
+  linkReceiverFactory: ContractFactory,
+  Token677ReceiverMockFactory: ContractFactory,
+  notERC677CompatibleFactory: ContractFactory,
+  extraPublicABI: string[],
+) => {
+  describe('ERC677Token', () => {
+    const provider = setup.provider()
+
+    // @ts-ignore
```

Comment:
We can use [this](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments) now :D